webpack/components/extensions/RegistrationCommands/fields/ActivationKeys.js

```diff
@@ -41,7 +41,12 @@ const ActivationKeys = ({
   useEffect(() => {
     handleInvalidField('Activation Keys', akHasValidValue(hostGroupId, pluginValues?.activationKeys, hostGroupActivationKeys));
   }, [handleInvalidField, hostGroupId, hostGroupActivationKeys, pluginValues]);
-
+  
+  useEffect(() => {
+    if(activationKeys.length==1){
+      onChange({ activationKeys: [activationKeys[0]?.name]})
```

Comment:
maybe you could reuse the `onSelect` function above instead by passing the value of the activation key?